Include/internal/pycore_condvar.h

```diff
@@ -31,7 +31,9 @@
 #define Py_HAVE_CONDVAR
 
 /* include windows if it hasn't been done before */
+#ifndef WIN32_LEAN_AND_MEAN
 #define WIN32_LEAN_AND_MEAN
```

Comment:
nitpick: can you add two spaces for readability? `#  define WIN32_LEAN_AND_MEAN`.